app/assets/javascripts/discourse/tests/acceptance/bootstrap-mode-notice-test.js

```diff
@@ -34,5 +35,12 @@ acceptance("Bootstrap Mode Notice", function (needs) {
       "/wizard/steps/hello-world",
       "it transitions to the wizard page"
     );
+
+    set(this.siteSettings, "bootstrap_mode_enabled", false);
+    await settled();
+    assert.ok(
```

Comment:
We're on the Wizard page at this point, right? So we would never expect the banner to show? I think we need to be on the homepage for this test to be meaningful?

```suggestion
    visit("/");
    set(this.siteSettings, "bootstrap_mode_enabled", false);
    await settled();
    assert.ok(
```

Reply:
That is absolutely correct! Pushed a fix. 👍🏼

Reply:
Don't you need to `await` the `visit("/")`?

Reply:
@davidtaylorhq does the `await settled()` takes care of `visit("/")` or should we use `await visit("/")` explicitly here?

Reply:
We should `await visit()` as well - my bad 👍